scipy/stats/_morestats.py

```diff
@@ -855,20 +855,20 @@ def ppcc_plot(x, a, b, dist='tukeylambda', plot=None, N=80):
     return svals, ppcc
 
 
-def _log_mean(logx):
+def _log_mean(logx, axis):
     # compute log of mean of x from log(x)
-    return special.logsumexp(logx, axis=0) - np.log(len(logx))
+    return special.logsumexp(logx, axis=axis) - np.log(len(logx))
 
 
-def _log_var(logx):
+def _log_var(logx, axis):
     # compute log of variance of x from log(x)
-    logmean = _log_mean(logx)
+    logmean = _log_mean(logx, axis=axis)
     pij = np.full_like(logx, np.pi * 1j, dtype=np.complex128)
-    logxmu = special.logsumexp([logx, logmean + pij], axis=0)
-    return np.real(special.logsumexp(2 * logxmu, axis=0)) - np.log(len(logx))
+    logxmu = special.logsumexp([logx, logmean + pij], axis=axis)
```

Comment:
```suggestion
    logxmu = special.logsumexp([logx, logmean + pij], axis=0)
```
I think this needs to stay at `0` actually because this is effectively subtracting the mean from each of the observations.